### PR TITLE
Update index.html etc. for #540 - Empty version directories must have inventories

### PR DESCRIPTION
**Pull request by neilsjefferies**

### CODE REVIEW

**Review by rosy1280 — DISMISSED** [review]

**Review by julianmorley — DISMISSED** [review]

File: draft/spec/index.html [hunk 1]
```diff
@@ -908,10 +908,13 @@ <h2>Version Inventory and Inventory Digest</h2>
             Every OCFL Object <span id="E063">MUST</span> have an inventory file within the OCFL Object Root,
             corresponding to the state of the OCFL Object at the current version. Additionally, every version directory
             <span id="W010">SHOULD</span> include an inventory file that is an <a href="#inventory">Inventory</a> of
-            all content for versions up to and including that particular version. Where an OCFL Object contains
-            <code>inventory.json</code> in version directories, the inventory file in the OCFL Object Root
-            <span id="E064">MUST</span> be the same as the file in the most recent version. See also requirements for
-            the corresponding <a href="#inventory-digest">Inventory Digest</a>.
+            all content for versions up to and including that particular version. In the case when no files have been
+            added or updated in a given version, resulting in an empty and therefore absent
```
[the remark this answers is not in the document]
Comment (neilsjefferies) [reply draft/spec/index.html]: Done

File: draft/spec/index.html [hunk 2]
```diff
@@ -908,10 +908,13 @@ <h2>Version Inventory and Inventory Digest</h2>
             Every OCFL Object <span id="E063">MUST</span> have an inventory file within the OCFL Object Root,
             corresponding to the state of the OCFL Object at the current version. Additionally, every version directory
             <span id="W010">SHOULD</span> include an inventory file that is an <a href="#inventory">Inventory</a> of
-            all content for versions up to and including that particular version. Where an OCFL Object contains
-            <code>inventory.json</code> in version directories, the inventory file in the OCFL Object Root
-            <span id="E064">MUST</span> be the same as the file in the most recent version. See also requirements for
-            the corresponding <a href="#inventory-digest">Inventory Digest</a>.
+            all content for versions up to and including that particular version. In the case when no files have been
+            added or updated in a given version, resulting in an empty and therefore absent 
+            <a href="#content-directory">"content" directory</a>, the version directory <span id="E109">MUST</span>
```
[the remark this answers is not in the document]
Comment (neilsjefferies) [reply draft/spec/index.html]: Done

File: draft/spec/index.html [hunk 3]
```diff
@@ -911,10 +911,13 @@ <h2>Version Inventory and Inventory Digest</h2>
             Every OCFL Object <span id="E063">MUST</span> have an inventory file within the OCFL Object Root,
             corresponding to the state of the OCFL Object at the current version. Additionally, every version directory
             <span id="W010">SHOULD</span> include an inventory file that is an <a href="#inventory">Inventory</a> of
-            all content for versions up to and including that particular version. Where an OCFL Object contains
-            <code>inventory.json</code> in version directories, the inventory file in the OCFL Object Root
-            <span id="E064">MUST</span> be the same as the file in the most recent version. See also requirements for
-            the corresponding <a href="#inventory-digest">Inventory Digest</a>.
+            all content for versions up to and including that particular version. In the case where no files have been
```
Comment (rosy1280) [inline draft/spec/index.html:914]: I think we could make this cleaner by starting a new paragraph at 

> In the case where.....

But I'm not going to hold up approval for that simple change.

**Review by rosy1280 — APPROVED** [review]

**Comment by neilsjefferies** [comment]
Cover this in version directory description instead (also allow .keep rather than forcing an inventory)

**Pull request closed without merging**